### ISSUE TITLE
Won't logout from another tab because of REFRESH_TOKEN

### ISSUE THREAD

**Issue by ludojmj** [AxaGuilDEv/react-oidc#948]
### Issue and Steps to Reproduce

```
export const configurationIdentityServer = {
    client_id: 'interactive.public.short',
    redirect_uri: window.location.origin + '/authentication/callback',
    silent_redirect_uri: window.location.origin + '/authentication/silent-callback',
    scope: 'openid profile email api offline_access',
    authority: 'https://demo.duendesoftware.com',
    refresh_time_before_tokens_expiration_in_second: 40,
    service_worker_relative_url: '/OidcServiceWorker.js',
    service_worker_only: false,
    monitor_session: true,
    extras: { youhou_demo: 'youhou' },
    token_renew_mode: TokenRenewMode.access_token_or_id_token_invalid,
};
```

1) Run the demo (react-oidc/packages/react -> http://localhost:4200/)
2) Log in
3) Open another tab and go to [duendesoftware](https://demo.duendesoftware.com/)
4) Log out current user from [duendesoftware](https://demo.duendesoftware.com/) tab
5) Come back to the demo tab (http://localhost:4200/)
6) The token on the demo tab is refreshed (so that the user is still connected) despite being logged out on [duendesoftware](https://demo.duendesoftware.com/) tab 
7) You can check this behaviour thanks to the "Default configuration Events" (see screenshot)

### Versions

6.14.1

### Screenshots

![image](https://user-images.githubusercontent.com/11465186/216831481-43e17d7b-de12-4acd-a446-96db4e2ce1b0.png)

#### Expected

The token shouldn't be refreshed when user's logged out from another tab.
It was OK with version in 6.12.1 but not anymore with version 6.14.1

#### Actual

The token keeps being refreshed despite logged out on another tab.

**Comment by guillaume-chervet** [comment]
Ho, thank you very much @ludojmj. I can reproduce it.
I fix it tommorrow afternoon.

**Comment by guillaume-chervet** [comment]
fixed by https://github.com/AxaGuilDEv/react-oidc/pull/952 @ludojmj

**Comment by guillaume-chervet** [comment]
I close the issue @ludojmj , thank you for it. Feel free to reopen it if you need.

**Comment by ludojmj** [comment]
fix(oidc): single logout #952 (https://github.com/AxaGuilDEv/react-oidc/pull/952)

FIXES https://github.com/AxaGuilDEv/react-oidc/issues/944 AND https://github.com/AxaGuilDEv/react-oidc/issues/947

BUT NOT THIS ACTUAL ISSUE! (https://github.com/AxaGuilDEv/react-oidc/issues/948)
